cardano-api/src/Cardano/Api/TxBody.hs

```diff
@@ -2035,7 +2038,7 @@ makeShelleyTransactionBody era@ShelleyBasedEraShelley
         scripts
         TxBodyNoScriptData
         txAuxData
-        txScriptValidity
+        (unBuildTxWith txScriptValidity)
```

Comment:
This should be hardcoded to `TxScriptValidityNone` as era before Alonzo have no notion of script validity.

Reply:
Done!